worker/src/main/java/org/apache/celeborn/service/deploy/worker/shuffledb/LevelDBProvider.java

```diff
@@ -93,6 +93,19 @@ public static org.iq80.leveldb.DB initLevelDB(File dbFile, StoreVersion version)
     return tmpDb;
   }
 
+  private static void createIfMissing(Options dbOptions, File dbFile) {
```

Comment:
This method looks similar to the same method in RocksDBProvider.

Reply:
@FMX, the `Options` class is different, therefore I did not create util method for this. Meanwhile, the initialization of database of `LevelDBProvider` and `RocksDBProvider` is mostly similar, which is better to create util method.